scripts/tests.e2e.sh

```diff
@@ -3,15 +3,15 @@ set -e
 
 export RUN_E2E="true"
 # e.g.,
-# ./scripts/tests.e2e.sh 1.9.5 1.9.4 0.4.7
+# ./scripts/tests.e2e.sh 1.9.7 1.9.6 0.4.8
```

Comment:
Maybe we don't need to always have to update these, as they are just examples. Alternatively move the `DEFAULT_` vars before this block, then we can also put

`# ./scripts/tests.e2e.sh $DEFAULT_VERSION1 $DEFAULT_VERSION2 $DEFAULT_SUBNET_EVM_VERSION` 

Just a suggestion